chapter/design-principles.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <div>
+        <nav>
+            <a href="../index.html">Navigation</a>
+        </nav>
+    </div>
```

Comment:
Wenn du `<nav>` verwendest, musst du den nicht unbedingt nochmal in einen `<div>` packen